<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deleting confirmation</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #8BC6E2;
}
QDialogButtonBox {
    background-color: #FFA500; /* Orange background color */
    border: 1px solid #FFA500; /* Orange border color */
    border-radius: 10px; /* Increased border corner radius for a rounder appearance */
    padding: 10px; /* Padding inside the box */
}

/* Style the &quot;Yes&quot; button */
QDialogButtonBox#dialog_yes_no QPushButton[role=&quot;Yes&quot;] {
    background-color: #FFA500; /* Orange background color */
    color: #000; /* Black text color */
    border: 1px solid #FFA500; /* Orange border color */
    border-radius: 15px; /* Increased border corner radius for a rounder appearance */
    padding: 10px 20px; /* Padding inside the button */
}

/* Style the &quot;No&quot; button */
QDialogButtonBox#dialog_yes_no QPushButton[role=&quot;No&quot;] {
    background-color: #FFA500; /* Orange background color */
    color: #000; /* Black text color */
    border: 1px solid #FFA500; /* Orange border color */
    border-radius: 15px; /* Increased border corner radius for a rounder appearance */
    padding: 10px 20px; /* Padding inside the button */
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="dialog_yes_no">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>2</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: orange;
        color: black;
        border-radius: 15px;
        padding: 10px 20px;
        font-size: 16px;
        font-family: &quot;Arial Narrow&quot;, sans-serif;
    }
    
    QPushButton:pressed {
        background-color: darkorange; /* Change color when button is pressed */
    }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #ff0000; /* Red text color */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff5856;&quot;&gt;Are you sure, you want to delete selected records?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_yes_no</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_yes_no</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
